Snowflake-Labs/semantic-model-generator
# Integrate builder flow into admin app

The current admin app only supports the "iteration" flow, which takes an existing semantic model and allows the user to append verified queries. We'd also like to integrate the "builder" flow (i.e. building a semantic model from scratch), which is essentially a wrapper around the existing CLI generator tool.

This PR fleshes out the admin app so that the user can create a YAML from scratch and proceed into the builder flow. We've also left in the option to jump directly to the iterator flow.

## Testing instructions

1. Follow setup instructions in the root README to install dependencies and get everything set up.
2. Run through the full builder and iteration flows separately to make sure everything works as intended, culminating in a final upload to the stage.
